speech/api/speech_streaming.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+
+import contextlib
+import threading
+
+from gcloud.credentials import get_credentials
+
+from google.cloud.speech.v1.cloud_speech_pb2 import *  # noqa
+from google.rpc import code_pb2
+
+from grpc.beta import implementations
+
+import pyaudio
+
+# Audio recording parameters
+RATE = 16000
+CHANNELS = 1
+CHUNK = RATE // 10  # 100ms
+
+# Keep the request alive for this many seconds
+DEADLINE_SECS = 8 * 60 * 60
+SPEECH_SCOPE = 'https://www.googleapis.com/auth/cloud-platform'
+
+
+def _make_channel(host, port):
```

Comment:
This functions aren't part of a module, so there's no need to make them private.